resources/services/iot/iot_streams.go

```diff
@@ -130,7 +131,7 @@ func fetchIotStreams(ctx context.Context, meta schema.ClientMeta, parent *schema
 			options.Region = c.Region
 		})
 		if err != nil {
-			return diag.WrapError(err)
+			return diags.Add(diag.FromError(diag.WrapError(err), diag.RESOLVING, diag.WithSeverity(diag.ERROR)))
```

Comment:
Diags are most prob going away but just checking that this is what is intended: `diags.Add, diag.FromError, diag.WrapError`